flink-end-to-end-tests/test-scripts/test_streaming_bucketing.sh

```diff
@@ -27,7 +27,7 @@ function get_total_number_of_valid_lines {
   # this method assumes that pending files contain valid data.
```

Comment:
The comment is now invalid as we do not assume that pending files have valid data anymore.